hack/csv-generate.sh

```diff
@@ -43,32 +51,47 @@ fi
 # clean up all old data first
 clean_package
 
+# do not generate new CRD for old versions
+if [[ "$CSV_VERSION" != "$LATEST" ]]; then
```

Comment:
when we have the 4.4.0 CRD in deploy/olm-catalog/performance-addon-operator/4.4.0 already, maybe do like `if version != prev then generate crds`? (otherwise nothing)